bindings/rust/aws-lc-fips-sys-template/Cargo.toml

```diff
@@ -0,0 +1,38 @@
+[package]
+name = "aws-lc-fips-sys"
+description = "AWS-LC is a general-purpose cryptographic library maintained by the AWS Cryptography team for AWS and their customers. This is the FIPS validated version of AWS-LC."
+version = "__AWS_LC_FIPS_SYS_VERSION__"
+authors = ["AWS-LC"]
+edition = "2021"
+repository = "https://github.com/awslabs/aws-lc"
+license = "Apache-2.0 OR ISC"
+rust-version = "1.57.0"
+include = [
+    "build.rs",
+    "Cargo.toml",
+    "deps/aws-lc/**/*.c",
```

Comment:
Missing `build_bindgen.rs`?

Reply:
This one was intentional. We don't expose the `generate_bindings` in `aws-lc-fips-sys` since FIPS is only supported on the platforms we offer pre-generated bindings for, so the `build_bindgen` file isn't needed in the official crate.